Specs/DataSources/PolygonGeometryUpdaterSpec.js

```diff
@@ -272,12 +277,14 @@ describe(
 
     it("Creates coplanar polygon", function () {
```

Comment:
I think the test below "dynamic updater sets properties" should be updated as well